src/utils/convertExperimentToApiModel.js

```diff
@@ -0,0 +1,20 @@
+// There are some differences between the property naming of elements
+// stored in the ui and in the api,
+// this is an attempt to deal with this in one single place (the ui)
+// We should try to converge to one single model to follow
```

Comment:
I agree. It will be best to not have to do such conversions. When do you think it will be a good time to do this conversion?

Reply:
Nevermind, I actually see that for the v2 version, the api uses the same property names and there is no need for conversion there. Sounds great!

Reply:
In this case, you can changes this comment to say that after we've completed the migration to api v2, the conversion will not be necessary